src/ralph/api/routers/statements.py

```diff
@@ -287,6 +300,8 @@ async def post(statements: Union[LaxStatement, list[LaxStatement]]):
             detail="Duplicate statement IDs in the list of statements",
         )
 
+    background_tasks.add_task(forward_xapi_statements, list(statements_dict.values()))
```

Comment:
In the first version of LearningLocker, the statement forwarding is done after the successful storage of the xAPI statement. In the current version of LearningLocker, it seems, the forwarding is done in parallel with validation & storage, though I'm not sure(
Which approach should we choose? Or maybe we should make it configurable? WDYT?